packages/beacon-node/src/sync/unknownBlock.ts

```diff
@@ -92,7 +92,8 @@ export class BlockInputSync {
     private readonly opts?: SyncOptions
   ) {
     this.maxPendingBlocks = opts?.maxPendingBlocks ?? MAX_PENDING_BLOCKS;
-    this.proposerBoostSecWindow = this.config.SECONDS_PER_SLOT / INTERVALS_PER_SLOT;
+    // TODO GLOAS: Pass in a real fork name
+    this.proposerBoostSecWindow = config.getAttestationDueMs(ForkName.phase0) / 1000;
```

Comment:
same here, it's not correct to set this in constructor anymore as we need those timings to be dynamic